packages/config/config/devices/0x0063/26932_26933_zw3008.json

```diff
@@ -24,12 +24,14 @@
 			"isLifeline": true
 		},
 		"2": {
-			"label": "Group 2",
-			"maxNodes": 5
+			"label": "Group 2 Basic Set",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled by pressing On/Off button with local load"
 		},
 		"3": {
-			"label": "Group 3",
-			"maxNodes": 5
+			"label": "Group 3 Basic Set",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled by pressing On/Off button with local load"
```

Comment:
What's the difference between groups 2 and 3 here? Or is this a 2-channel device?

Reply:
Must be dual channel, documentation attached, do you want me to just use one? Or leave both?
![951523AF-F5FA-4181-8087-5EDF9D04E724](https://user-images.githubusercontent.com/24241104/121807472-4e53c800-cc22-11eb-9ffd-e8fed8dcdd24.jpeg)

Reply:
Might also be On-Button => Group 2 and Off-Button => Group 3. Let's see if we can find someone to confirm.